public/app/features/alerting/unified/utils/k8s/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { config } from '@grafana/runtime';
```

Comment:
(Once https://github.com/grafana/grafana/pull/90094 is merged, I will consolidate these together)